src/xmipp/libraries/reconstruction/nma_alignment_vol.cpp

```diff
@@ -224,10 +237,15 @@ double ObjFunc_nma_alignment_vol::eval(Vector X, int *nerror) {
 
 	FileName fnRandom = global_nma_vol_prog->createDeformedPDB();
 	const char * randStr = fnRandom.c_str();
+	FileName fnShiftsAngles = formatString("%s_angles_shifts.txt", randStr);
+	const char * shifts_angles = fnShiftsAngles.c_str();
```

Comment:
This is already getting confusing. 
Isn't it simply `fnRandom + "_angles_shifts.txt"`?